src/main/java/com/parasoft/parabank/service/ParaBankServiceImpl.java

```diff
@@ -499,4 +500,10 @@ public String withdraw(final int accountId, final BigDecimal amount) throws Para
             throw new ParaBankServiceException("Could not find account number " + accountId, e);
         }
     }
+
+    @Override
+    public String billPay(int accountId, BigDecimal amount, Payee Payee) throws ParaBankServiceException {
```

Comment:
Minor feedback:  I notice that all the methods in this class were declared in alphabetical order before you added billPay.